### ISSUE TITLE
RPi3 build from master successfully but won't boot.

### ISSUE THREAD

**Issue by sghazagh** [OpenELEC/OpenELEC.tv#4868]
Hi all,
I tried to compile the Master branch for **Raspberry Pi3** from fresh copy of OpenELEC GIT and the compile has done successfully.
However, I saw the boot screen and it probably done the partition resize but in the next reboot it just shows the black screen and does not go through the boot!
Anyone knows how to fix it?

Also, which branch is the 6.0.3 which has been compiled and published for RPi3? Should I compile Master if I wanna have the same version or there is another branch for that?

Can anybody help please?

**Comment by sraue** [comment]
we need more infos about the problems, you are able to connect via ssh to the box? and get some logs?

**Comment by sghazagh** [comment]
Hi Stephan,
I couldn't get access through SSH but I had another application which I somehow could get the file out of the system.
In fact it's some sort of network browsing application!!!
Anyway, I could get the file named "log-2016-03-23-09.15.59.zip" which contains all the log files.
I can see from one of the log files that says:
**09:15:40 296.214508 T:1963188224   FATAL: addon 'service.xbmc.versioncheck' is missing.
09:15:40 296.214600 T:1963188224   FATAL: CServiceManager::Init: Unable to start CAddonMgr**

I don't know if it's really a valid error or not!
I have uploaded the zipped log file on my Google Drive and here is the link to download:
https://drive.google.com/file/d/0B8w4ZQV13bCSdXFya0xuRVphbzA/view?usp=sharing

I appreciate if you can have a look and see why the master still does not work and assist me to resolve this issue.

Many thanks,

**Comment by sraue** [comment]
thanks much, can you test a build from latest master again? can you before(!!!!) provide or paste the content from /usr/share/kodi/system/addon-manifest.xml from the affected system?

**Comment by sghazagh** [comment]
Hi,
Here is the content of the "**addon-manifest.xml**" file.

Just a note, I have extracted this file from "**OpenELEC-RPi2.arm-8.0-devel-20160322170607-r22464-g2afa5c1.system**" file located in "**target**" folder by mounting the file.
I assume it will be similar to the one which mounts on OpenELEC run,correct?
If not, let me know and I will get it out from the affected system itself.

```
<addons>
  <addon>audioencoder.xbmc.builtin.aac</addon>
  <addon>audioencoder.xbmc.builtin.wma</addon>
  <addon>game.controller.default</addon>
  <addon>kodi.adsp</addon>
  <addon>kodi.audiodecoder</addon>
  <addon>kodi.game</addon>
  <addon>kodi.guilib</addon>
  <addon>kodi.inputstream</addon>
  <addon>kodi.peripheral</addon>
  <addon>kodi.resource</addon>
  <addon>metadata.album.universal</addon>
  <addon>metadata.artists.universal</addon>
  <addon>metadata.common.allmusic.com</addon>
  <addon>metadata.common.fanart.tv</addon>
  <addon>metadata.common.htbackdrops.com</addon>
  <addon>metadata.common.imdb.com</addon>
  <addon>metadata.common.musicbrainz.org</addon>
  <addon>metadata.common.theaudiodb.com</addon>
  <addon>metadata.common.themoviedb.org</addon>
  <addon>metadata.local</addon>
  <addon>metadata.themoviedb.org</addon>
  <addon>metadata.tvdb.com</addon>
  <addon>repository.xbmc.org</addon>
  <addon>resource.images.weathericons.default</addon>
  <addon>resource.language.en_gb</addon>
  <addon>resource.uisounds.confluence</addon>
  <addon>screensaver.xbmc.builtin.black</addon>
  <addon>screensaver.xbmc.builtin.dim</addon>
  <addon>script.module.pil</addon>
  <addon>service.xbmc.versioncheck</addon>
  <addon>skin.estuary</addon>
  <addon>skin.estouchy</addon>
  <addon>webinterface.default</addon>
  <addon>xbmc.addon</addon>
  <addon>xbmc.audioencoder</addon>
  <addon>xbmc.codec</addon>
  <addon>xbmc.core</addon>
  <addon>xbmc.gui</addon>
  <addon>xbmc.json</addon>
  <addon>xbmc.metadata</addon>
  <addon>xbmc.pvr</addon>
  <addon>xbmc.python</addon>
  <addon>xbmc.webinterface</addon>
</addons>
```
I am also compiling from the master again. will post the result here...

Thanks for your assistance.

**Comment by sraue** [comment]
yes thats exactly what i thougt... the new one should be dinfferent, and that was the problem :-) thanks much. please report back with the new file and if it works now

**Comment by sghazagh** [comment]
Hi Stephan,
Yes, the content of the new compile is different.

![compare](https://cloud.githubusercontent.com/assets/12246858/13999878/9281c2ae-f192-11e5-953b-55ad3382978d.png)

However, I cannot test the actual RPi3 image as I am far from home till afternoon.
Will test the created image on RPi3 and will update the result here.
Thanks for your help

**Comment by sraue** [comment]
thats exactly the problem, which should be fixed now :-) thanks much again, please close this ticket when you dont get the error from above anymore

**Comment by sghazagh** [comment]
Stephan,
I came home and first thing I did was writing the image into a SD card and tried to see if the new compile works fine or not.
Unfortunately, the image still has the same issue. It shows the black screen and does not load to KODI environment.

Here is the new log file for my latest build:
https://drive.google.com/file/d/0B8w4ZQV13bCSaGJ1Z1ZuclF2eW8/view?usp=sharing

Hope you can find what the problem is.
Many thanks,

**Comment by sghazagh** [comment]
Anyone could fix this issue?
